### ISSUE TITLE
Simple Voice Chat Support

### ISSUE THREAD

**Issue by Sammebamme123** [mt1006/mc-mocap-mod#41]
It would be so fun to have voice lines using voice chat! Would this be possible, or is this something the Simple Voice Chat team needs to add?

**Comment by mt1006** [comment]
Yes, it would be possible. I have plans to add ability to give in-game voices for played players, but it probably won't be in the 1.4 version, but rather more distant future.
